src/main/javascript/email_settings.js

```diff
@@ -33,6 +33,7 @@ const EmailSettingsTableUser = ({ isAdmin }) => {
             url: '/protected/update_email_preferences',
             data: submissionData,
             dataType: 'json',
+            async: false,
```

Comment:
Is this to make sure the buttons don't get fiddled with until the preferences have been updated in the backend?